src/components/visualizations/chart/highcharts/plugins/test/getTickPositions.spec.ts

```diff
@@ -611,17 +611,14 @@ describe('get tick positions for dual axes chart', () => {
     });
 
     describe('createArrayFromRangeWithMiddleZero', () => {
-
-        const runTest = (start: number, end: number, ticksNumber: number = TICK_AMOUNT) => {
+        it.each([
+            [-100000, 1000000, 12],
+            [-0.2100000000000003, 2.1, 12]
+        ])('should add zero between start and end values', (start: number, end: number, ticksNumber: number) => {
```

Comment:
you can even use `start` and `end` values in test description, so it is possible to distinguish which of two runs failed